src/data/downloads/links.yml

```diff
@@ -20,6 +20,14 @@ decrediton:
     - title: Release notes
       url: https://github.com/decred/decred-binaries/blob/master/release-notes.md#decrediton-v140
 
+mobilewallets:
+    - title: Android
+      url: https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet
+      is_direct_download: true
```

Comment:
These are not direct download links, they are links to the play/app store

Reply:
I know but arrow points to the right then is that okay?

Reply:
I think so yes, lets keep it consistent with the rest of the page.